### DISCUSSION TITLE
WiFi Pineapple Mark VII support and testing

### DISCUSSION THREAD

**Discussion by kleo** [kbeflo/evilportal-templates#41] [General]
Hello everyone! I haven't got a WiFi Pineapple Mark VII in hand for testing evilportals. We could collaborate here for compatibility issues and testing.

**Comment by jethr0-1** [comment]
Hey there! I'm not sure if it's a compatibility issue or not, but I modified a portal template and removed the email part of the HTML/PHP form, everything else should be unmodified. Can this cause issues? If so, how can I make it so that an evil portal only has a password form? Thanks! 👍

**Reply by kleo** [reply]
You should cleanup MyPortal.php fields too to avoid blank fields on your logs

**Reply by jethr0-1** [reply]
Is that the only other file I would need to modify? I just want to remove the email field entirely

**Reply by kleo** [reply]
Yes

**Reply by jethr0-1** [reply]
Hey, I'm modifying the files now but I've noticed I think the MyPortal.php file relies on the email parameter "if (isset($_POST['email']))". If I were to change this to $_POST['password'] would the portal work properly?

**Reply by syed-shirazi** [reply]
can you show us your new login portal please ? it may help others too thanks 👍

**Comment by kleo** [comment]
I found a reference video on YouTube on setting up on the WiFi Pineapple MK VII. Can someone try out his steps on setting up. Video by  [Black Cat](https://www.youtube.com/channel/UC5QAncbbsunhSxINE2Vqyig) https://www.youtube.com/watch?v=9oo7IfDov-Q

**Reply by jethr0-1** [reply]
Yep this video is correct and works fine.

**Comment by syed-shirazi** [comment]
Hi everyone, can someone explain me please, once we have clicked on the button login on a evil portal, we are redirected to a page, it's written " you have not been authorized " .
I would like to know if we can modify it, and customize it.
Thanks

**Comment by SgtFoose** [comment]
Hi Kleo, I'm using your portals on my MK7 v2.1.3 stable. When the MK7 is untouched / freshly installed, and I hit add portal (basic or targeted), it adds the default portals as expected, but when I upload yours they will always appear as BASIC instead of TARGETED. Therefore, the Preview and View Log buttons won't show. Your portals do actually run with no problem. I will contact newbie to see what the problem is. One thing I noticed, is that your CSS bootstrap.min.cs file from the Starbucks Portal contains quite a few errors which I've managed to resolve. I cant upload a .css file here so I put it in a txt file.

**Comment by SgtFoose** [comment]
[bootstrap.min.txt](https://github.com/user-attachments/files/19182859/bootstrap.min.txt)

**Comment by PEmomenball** [comment]
I am using EvilPortal on an MK7.  When connecting to the openwifi from my phone I get the portal as i should.  I enter info into the form and submit. I get "access not authorized".  When I try to view the log for the portal though it shows nothing at all.  What am I doing wrong?

**Reply by SgtFoose** [reply]
Nothing wrong, the whole purpose is so people keep entering trying their know usernames and passwords

**Reply by PEmomenball** [reply]
It when I click on the log button I should see the info they entered?  The log is showing as blank still
